app/models/app.rb

```diff
@@ -19,9 +19,9 @@ class App
   # Some legacy apps may have string as key instead of BSON::ObjectID
   # identity :type => String
   field :_id,
-    type: String,
-    pre_processed: true,
-    default: ->{ BSON::ObjectId.new.to_s }
+        type: String,
+        pre_processed: true,
+        default: ->{ BSON::ObjectId.new.to_s }
```

Comment:
This is one cop where I've run into trouble before. Maybe I have to find a way to get used to this, but the previous incantation looks more natural to me.